grammars/rust.cson

```diff
@@ -142,6 +142,11 @@
     'name': 'support.type.std.rust'
     'match': '\\b(ToOwned|ToString)\\b'
   }
+  'type': {
+    'comment': 'A type'
+    'name': 'entity.name.type.rust'
+    'match': '\\b[A-Za-z0-9_]+\\b'
```

Comment:
It is wrong regex because it matches for `_`, `0` and etc.

The correct is `\\b([A-Za-z][A-Za-z0-9_]*|_[A-Za-z][A-Za-z0-9_]*)\\b`.